Sources/SwiftParser/Declarations.swift

```diff
@@ -330,6 +331,8 @@ extension Parser {
       case `var`
       case `let`
       case `func`
+      case `borrow`
+      case `inout`
```

Comment:
Could you name the cases the same as the keyword? We aren’t doing it everywhere right now but it’s something I’d like to change when I’ve got time.